angular/projects/admin-nrpti/src/app/records/agreements/agreement-add-edit/agreement-add-edit.component.html

```diff
@@ -121,8 +121,15 @@ <h4 class="blue-header mb-0">LNG Site Content</h4>
           <span class="grey-subtext-no-margins">{{ lngPublishSubtext }}</span>
         </div>
         <div class="col-4">
-          <mat-slide-toggle (toggleChange)="togglePublish('lng')" [checked]="lngPublishStatus === 'Published'">
-            {{ lngPublishStatus }}
+          <mat-slide-toggle
+            name="publishLng"
+            id="publishLng"
+            formControlName="publishLng"
```

Comment:
Missing `class="form-control"`

I don't actually know what this class does specifically, but everything else has it, so presumably its important.